mods/d2k/maps/atreides-01a/atreides01a.lua

```diff
@@ -71,21 +72,21 @@ Tick = function()
 	-- player has no Wind Trap
 	if (player.PowerProvided <= 20 or player.PowerState ~= "Normal") and DateTime.GameTime % DateTime.Seconds(32) == 0 then
 		HasPower = false
-		Media.DisplayMessage(Messages[2], "Mentat")
+		Media.DisplayMessage(Messages[2], Mentat)
 	else
 		HasPower = true
 	end
 
 	-- player has no Refinery and no Silos
 	if HasPower and player.ResourceCapacity == 0 and DateTime.GameTime % DateTime.Seconds(32) == 0 then
-		Media.DisplayMessage(Messages[3], "Mentat")
+		Media.DisplayMessage(Messages[3], Mentat)
 	end
 
 	if HasPower and player.Resources > player.ResourceCapacity * 0.8 and DateTime.GameTime % DateTime.Seconds(32) == 0 then
-		Media.DisplayMessage(Messages[4], "Mentat")
+		Media.DisplayMessage(Messages[4], Mentat)
 	end
 
-	UserInterface.SetMissionText("Harvested resources: " .. player.Resources .. "/" .. SpiceToHarvest, player.Color)
+	UserInterface.SetMissionText(HarvestedResources .. player.Resources .. "/" .. SpiceToHarvest, player.Color)
```

Comment:
We made this use translation parameters in the RA PR, didn't we?

Reply:
Added cached lookups like in https://github.com/OpenRA/OpenRA/pull/20387/.